sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -20,34 +21,45 @@ object AkkaGrpcPlugin extends AutoPlugin {
   override def requires = ProtocPlugin
 
   trait Keys { _: autoImport.type =>
+
+    object AkkaGrpc {
+      sealed trait TargetStub
```

Comment:
I'm not sure I like the `stub` term, it seems to be a bit overloaded.. perhaps just `Target`?

Reply:
I wanted to avoid just `Target` alone since `Target` is the end result of this exercise.  
`GeneratedSources`? (a bit too narrow perhaps, since that also includes the model which isn't configurable)

Reply:
IMO  `stub` is only extensively used in the gRPC context therefore it felt natural as the common name of the client and server. Only having `Target` overloads it as we also have `TargetLanguage`.

Reply:
right, but gRPC uses 'stub' mainly to describe the client AFAICS (https://grpc.io/docs/guides/concepts.html), it might in fact be confusing to also use it for other generated code?

I agree that just `Target` isn't very nice either though ;). `GeneratedSources` might be fine, and then in the setting comment explain it is 'which sources to generate in addition to the gRPC protobuf model classes'?